libcodechecker/server/client_db_access_server.py

```diff
@@ -585,6 +586,22 @@ def __init__(self,
             HTTPServer.__init__(self, server_address,
                                 RequestHandlerClass,
                                 bind_and_activate=True)
+            ssl_key_file = os.path.join(config_directory, "key.pem")
+            ssl_cert_file = os.path.join(config_directory, "cert.pem")
+            if os.path.isfile(ssl_key_file) and os.path.isfile(ssl_cert_file):
+                LOG.info("Initiating SSL. Server listening on secure socket.")
+                LOG.debug("Using cert file:"+ssl_cert_file)
+                LOG.debug("Using key file:"+ssl_key_file)
+                self.socket = ssl.wrap_socket(self.socket, server_side=True,
+                                              keyfile=ssl_key_file,
+                                              certfile=ssl_cert_file)
+
+            else:
+                LOG.info("SSL key or cert files not found. cert:" +
```

Comment:
Do we want to display this message even if the user never configured this? It is just an ergonomic change in the future.